src/code-editor/code-editor.component.ts

```diff
@@ -558,6 +589,7 @@ export class TdCodeEditorComponent implements OnInit, AfterViewInit, ControlValu
         value: this._value,
         language: this.language,
         theme: this._theme,
+        automaticLayout: this._automaticLayout,
```

Comment:
cool, if thats the case should we make this an input without value?

```
<td-code-editor automaticLayout style="height: 200px" theme="vs" flex language="javascript" [(ngModel)]='frontEndJSCode' id="js"></td-code-editor> 
```

So its either there or not 😄